python/ray/_private/utils.py

```diff
@@ -302,6 +302,9 @@ def writelines(self, datas):
         self.stream.flush()
 
     def __getattr__(self, attr):
+        # Avoid endless loop when get `stream` attribtue
```

Comment:
Could you add some tests?

Reply:
this bug can simply be reprduced, a local test if enough i think, 
<img width="1349" height="1038" alt="image" src="https://github.com/user-attachments/assets/9124ac40-f1cd-4a21-aac9-784e7d716db5" />

Reply:
Actually why do you want to pickle Unbuffered?

Reply:
You can just have a test that pickle and unpickle Unbuffered and make sure it works.